added Fritzbox_WiFi switch (#7935)
home-assistant/home-assistant.github.io: source/_components/switch.fritzbox_wifi.markdown

---
layout: page
title: "AVM FRITZBOX (Guest) WIFI Switch"
description: "Instructions on how to integrate your AVM FRITZBOX (Guest) WIFI as switches into Home Assistant."
date: 2018-12-23 17:00
sidebar: true
comments: false
sharing: true
footer: true
logo: avm.png
ha_category: Switch
ha_release: ""
ha_iot_class: "Local Polling"
---
{% linkable_title FRITZBOX WiFi Switch %}

The `fritzbox_wifi` switch allows you to switch on/off your AVM FRITZBOX 2,4GHz/5GHz/Guest-WiFi.

> If you have more then one Fritzbox, multiple Fritzboxes ar supported.

Reviewer: ```suggestion
If you have more than one Fritzbox, multiple Fritzboxes ar supported.
```

> If you have more than one Fritzbox, multiple Fritzboxes ar supported.

Reviewer 2: Fritzboxes are supported

> If you want to controll 2,4GHz an Guest-Wifi setup 2 switches (and so on)...

Reviewer: ```suggestion
If you want to control 2,4GHz and Guest-Wifi setup 2 switches (and so on)...
```

> 

Reviewer 2: Stray blank line

To use your AVM FRITZBOX WiFi switch(es) in your installation, add the following to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry
switch:
  - platform: fritzbox_wifi
    host: 192.168.XXX.XXX
    password: "YOUR_PASSWORD"
```

{% configuration %}
host:

>   description: The IP for your Fritz!Box.

Reviewer 2: Description says IP, default is not an IP, some clarity is required ;)

  required: true
  type: string
  default: fritz.box
password:
  description: The password for your Fritz!Box.
  required: true
  type: string
name:
  description: The switch name in your HA installtion.
  required: false
  type: string
  default: "Guest WiFi"
username:
  description: The username of your Fritzbox Login of you have.
  required: false
  type: string
interface:

>   description: "The WiFi interface you want to control. Most modern Fritzboxes have 3 Network-Interfaces: 2,4GHz -> 1, 5GHz -> 2, Guest Wifi -> 3. If you have no 5GHz, Guest Wifi should be interface: 2."

Reviewer: ```suggestion
  description: "The WiFi interface you want to control. Most modern Fritzboxes have 3 network interfaces: 2,4GHz -> 1, 5GHz -> 2, Guest Wifi -> 3. If you have no 5GHz, Guest Wifi should be interface: 2."
```

---
layout: page
title: "AVM FRITZBOX (Guest) WIFI Switch"
description: "Instructions on how to integrate your AVM FRITZBOX (Guest) WIFI as switches into Home Assistant."
date: 2018-12-23 17:00
sidebar: true
comments: false
sharing: true
footer: true
logo: avm.png
ha_category: Switch
ha_iot_class: "Local Polling"

> ---

Reviewer 3: Missing: `ha_release:`

Author: what should I fill out there if its not released?

Reviewer 3: You can leave it blank until you know in which release it will be merged. Or fill in 0.85.0 if it is likely that the PR of the component will be quickly merged before the next release in Januari.

> {% linkable_title %}

Reviewer 3: It is missing a title name

{% linkable_title FRITZBOX WiFi Switch %}

The `fritzbox_wifi` switch allows you to switch on/off your AVM FRITZBOX 2,4GHz/5GHz/Guest-WiFi.

If you have more then one Fritzbox, multiple Fritzboxes ar supported.
If you want to controll 2,4GHz an Guest-Wifi setup 2 switches (and so on)...

To use your AVM FRITZBOX WiFi switch(es) in your installation, add the following to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry
switch:
  - platform: fritzbox_wifi
    host: 192.168.XXX.XXX
    password: "YOUR_PASSWORD"

>     name: "NAME_OF_THE_SWITCH_SHOW_UP_IN_HA"

Reviewer 4: Please remove optional options from the example, this should only contain the minimum (required) options to get the platform working.

```

{% configuration %}
host:
  description: The IP for your Fritz!Box.
  required: false
  type: string
  default: fritz.box
password:
  description: The password for your Fritz!Box.
  required: true
  type: string
name:
  description: The switch name in your HA installtion.
  required: false
  type: string
  default: "Guest WiFi"
username:
  description: The username of your Fritzbox Login of you have.
  required: false
  type: string
interface:

>   description: The WiFi interface you want to control. Most modern Fritzboxes have 3 Network-Interfaces:

Reviewer 3: ```suggestion
  description: "The WiFi interface you want to control. Most modern Fritzboxes have 3 Network-Interfaces: 2,4GHz -> 1, 5GHz -> 2, Guest Wifi -> 3. If you have no 5GHz, Guest Wifi should be interface: 2."
```

Reviewer 3: This will solve the build error

---
layout: page
title: "AVM FRITZBOX (Guest) WIFI Switch"
description: "Instructions on how to integrate your AVM FRITZBOX (Guest) WIFI as switches into Home Assistant."
date: 2018-12-23 17:00
sidebar: true
comments: false
sharing: true
footer: true
logo: avm.png
ha_category: Switch
ha_release:
ha_iot_class: "Local Polling"
---

> {% FRITZBOX WiFi Switch %}

Reviewer 3: ```suggestion
{% linkable_title FRITZBOX WiFi Switch %}
```

The line was correct only missing a name as title 😉

Author: 😂🙈

If you have more then one Fritzbox, multiple Fritzboxes ar supported.
If you want to controll 2,4GHz an Guest-Wifi setup 2 switches (and so on)...

To use your AVM FRITZBOX WiFi switch(es) in your installation, add the following to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry
switch:
  - platform: fritzbox_wifi
    host: 192.168.XXX.XXX
    password: "YOUR_PASSWORD"
    name: "NAME_OF_THE_SWITCH_SHOW_UP_IN_HA"
    interface: 3
```

{% configuration %}
host:
  description: The IP for your Fritz!Box.
  required: false
  type: string
  default: fritz.box
password:
  description: The password for your Fritz!Box.
  required: true
  type: string
name:
  description: The switch name in your HA installtion.
  required: false
  type: string
  default: "Guest WiFi"
username:
  description: The username of your Fritzbox Login of you have.
  required: false
  type: string
interface:
  description: "The WiFi interface you want to control. Most modern Fritzboxes have 3 Network-Interfaces: 2,4GHz -> 1, 5GHz -> 2, Guest Wifi -> 3. If you have no 5GHz, Guest Wifi should be interface: 2."

>   2,4GHz -> 1, 5GHz -> 2, Guest Wifi -> 3. If you have no 5GHz, Guest Wifi should be interface:2.

Reviewer 3: You can remove this line, it is now already in the description section.
